Lorem Ipsum XR Translator/Assets/Scripts/Translator/AzureTranslator.cs

```diff
@@ -0,0 +1,111 @@
+    {
+        string url = endpoint + string.Format("from={0}", from);
+        for (int i = 0; i < to.Length; i ++)
+        {
+            url += string.Format("&to={0}", to[i]);
+        }
+        Debug.Log(url);
+
+        // Create the request body
+        string requestBody = "[{ \"Text\": \"" + originalText + "\" }]";
+        byte[] requestData = System.Text.Encoding.UTF8.GetBytes(requestBody);
+
+        
+    
+        // Create a UnityWebRequest object
+        UnityWebRequest request = UnityWebRequest.Post(url, requestBody);
+        request.uploadHandler = new UploadHandlerRaw(requestData);
+    
+        // Set the request headers
+        request.SetRequestHeader("Content-Type", "application/json");
+        
+
+        request.SetRequestHeader("Ocp-Apim-Subscription-Key", this.apiKey);
+        request.SetRequestHeader("Ocp-Apim-Subscription-Region",this.location); 
+
+
+        request.downloadHandler = (DownloadHandler)new DownloadHandlerBuffer();
+    
+        // Send the request
+        yield return request.SendWebRequest();
+    
+        // Handle the response
+        if (request.result != UnityWebRequest.Result.Success)
+        {
+            Debug.Log("Error: " + request.error);
+            callback("Error: " + request.error);
+        }
+        else
+        {
+            string responseText = request.downloadHandler.text;
+            // Parse and process the response as needed
+            Debug.Log("Translation response: " + responseText);
+            callback(responseText);
+        }
+    }
+
+    //public static async Task Main()
+    //{
+        // Input and output languages are defined as parameters.
+    //    string route = "/translate?api-version=3.0&from=en&to=lan1&to=ja";
+    //    string textToTranslate = "Hi , my name is tintin";
+         
+    //    object[] body = new object[] { new { Text = textToTranslate } };
+    //    var requestBody = JsonConvert.SerializeObject(body);
+
+    //    using (var client = new HttpClient())
+    //    using (var request = new HttpRequestMessage())
+    //    {
+            // Build the request.
+    //        request.Method = HttpMethod.Post;
+    //        request.RequestUri = new Uri(endpoint + route);
+    //        request.Content = new StringContent(requestBody, Encoding.UTF8, "application/json");
+    //        request.Headers.Add("Ocp-Apim-Subscription-Key", key);
+            // location required if you're using a multi-service or regional (not global) resource.
+    //        request.Headers.Add("Ocp-Apim-Subscription-Region", location);
+
+            // Send the request and get response.
+    //        HttpResponseMessage response = await client.SendAsync(request).ConfigureAwait(false);
+            // Read response as a string.
+    //        string result = await response.Content.ReadAsStringAsync();
+           // Console.WriteLine(result);
+    //        Debug.Log(result);
+    //    }
```

Comment:
it might be a good idea to clean up this file a little bit by deleting commented out code

Reply:
and line 17, 22 and 28

Reply:
@VijaydityaSarker